staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -353,12 +356,9 @@ func (s preparedGenericAPIServer) Run(stopCh <-chan struct{}) error {
 		// and stop sending traffic to this server.
 		shutdownInitiatedCh.Signal()
 		klog.V(1).InfoS("[graceful-termination] shutdown event", "name", shutdownInitiatedCh.Name())
-
-		s.Eventf(corev1.EventTypeNormal, "TerminationStart", "Received signal to terminate, becoming unready, but keeping serving")
+		s.Eventf(corev1.EventTypeNormal, shutdownInitiatedCh.Name(), "Received signal to terminate, becoming unready, but keeping serving")
 
 		time.Sleep(s.ShutdownDelayDuration)
-
-		s.Eventf(corev1.EventTypeNormal, "TerminationMinimalShutdownDurationFinished", "The minimal shutdown duration of %v finished", s.ShutdownDelayDuration)
```

Comment:
have we lost this one?

Reply:
no, this is here - https://github.com/openshift/kubernetes/pull/896/files#diff-68eb390e148644c38174960be4d103c0e377e71eb540aa8c395f7d2278363973R349